### ISSUE TITLE
improve efficiency for reviewing studies/experiments/signatures

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/BugSigDB#131]
Are there any ways that the process of reviewing everything in a study could be made more efficient? I just reviewed a big study with 10 experiments (https://bugsigdb.org/Study_403#), and each time I click "Mark reviewed" it takes me back to the top of the page so I have to scroll down again. Leaving the reviewer's focus unchanged when clicking this button would make this process easier.

**Comment by tosfos** [comment]
We can prevent the page from reloading after a review. It would show green text indicating that an edit was made to mark it as reviewed. Please confirm.

**Comment by lwaldron** [comment]
That sounds great.

**Comment by ftzohra22** [comment]
@tosfos this is still happening where we click "Mark reviewed" and it takes us back to the top of the page.

**Comment by tosfos** [comment]
We checked into how to fix the issue but it's more of a fundamental issue with how the feature works. We're still researching it.

**Comment by lwaldron** [comment]
One thought for a workaround to make reviewing more efficient, how about a button "Mark all as reviewed" that marks the study and each experiment and signature within as reviewed?

**Comment by tosfos** [comment]
That would be difficult. We have a solution in action on the test site that we'll apply to production soon. It avoids the page reloading and also tries to not scroll the page after the button gets clicked.

**Comment by tosfos** [comment]
I thought about other possible solutions to allow "mass review" of pages. I feel like it might encourage low-quality reviews if users could just click a single button and mark many items as "reviewed". What do you think?

**Comment by lwaldron** [comment]
Agreed that "mass review" could encourage low-quality reviews, and avoiding page reloading and scrolling is a better solution, letting the reviewer mark items reviewed as they are done.

**Comment by tosfos** [comment]
I believe this can be closed.
